Hestia/app/src/main/java/hestia/UI/DeviceListActivity.java

```diff
@@ -16,9 +16,9 @@
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
-        ClientInteractionController cic = ClientInteractionController.getInstance();
+        BackendInteractor backendInteractor = BackendInteractor.getInstance();
         SharedPreferences prefs = getSharedPreferences("HESTIA.IP", 0);
-        cic.setIp(prefs.getString("IP_OF_SERVER", cic.getIp()));
+        backendInteractor.setIp(prefs.getString("IP_OF_SERVER", backendInteractor.getIp()));
```

Comment:
ditto, in general: no hardcoded constants.
UI: strings xml
code specific: static constant var to some class

This allows you to easily implement translations to different languages, renaming of items, and quick fixes.